eldritch/events.py

```diff
@@ -1471,8 +1490,10 @@ def resolve(self, state):
       return False
     if self.check.successes >= 1:
       self.evaded = True
+      self.character.avoid_monsters = True
```

Comment:
Upon further review, I definitely think this belongs in EvadeOrFightAll - there are some monsters where you can lose combat against them instead of successfully evading them.